File: Java_Practice/Java/loops4.java
// Sum of first N natural numbers (with series pattern)
import java.util.Scanner;

public class loops4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();
        int sum = 0;

        for (int i = 1; i <= n; i++) {
            sum += i;
            if (i < n) {
                System.out.print(i + " + ");
            } else {
                System.out.print(i + " = ");
            }
        }
        System.out.println(sum);
    }
}

File: Java_Practice/CPP/loops4.cpp
// sum of first N natural numbers
#include<iostream>
using namespace std;
int main()
{
    int n, sum  = 0;
    cout<<"ENTER N : ";
    cin>>n;
    for(int i = 1; i<=n; i++){
        sum += i;

        cout<<"sum of first N natural numbers are : "<<sum<<endl;

    }
    return 0;
    
}